modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -39,7 +41,7 @@ The Mule runtime engine passed functional testing against the following software
 |===
 |Software |Version
 | OS | MacOS 10.15, HP-UX 11i V3, AIX 7.2, Windows Server 2019, Windows 10, Solaris 11.3, Red Hat Enterprise Linux 8.8, Ubuntu Server 20.04
-| JDK | JDK 1.8.0, JDK 11, JDK 17
+| JDK | JDK 1.8.0, JDK 11, JDK 17, Adoptium OpenJDK distribution
```

Comment:
misleading. This seems to suggest that customers can only use this one, when the actual intent is to say that this is the one we use to test against.

Reply:
right, the java specification versions are already listed above